pod/perlsec.pod

```diff
@@ -268,6 +268,10 @@ problem will be reported:
 
   Insecure dependency in require while running with -T switch
 
+On versions of Perl before 5.26, activating taint mode will also remove
```

Comment:
past tense? "would also remove"

Reply:
I think what I have is correct. If you're using an older version (and many people are) then it will continue to act in this way.